selfdrive/test/test_car_models.py

```diff
@@ -313,6 +313,15 @@ def get_route_logs(route_name):
     'carFingerprint': TOYOTA.LEXUS_RXH,
     'enableCamera': True,
     'enableDsu': True,
+  },
```

Comment:
Think something went wrong solving the merge conflict here. You're adding the RX_TSS2 again, which was added in another PR.

Reply:
I was confused while rebasing, sorry.